app/components/PeopleItem/index.jsx

```diff
@@ -5,18 +5,26 @@ export default class PeopleItem extends React.Component {
   static propTypes = {
     item: PropTypes.object,
     isOnline: PropTypes.bool,
-    key: React.PropTypes.number,
   };
 
   render() {
     const offline = this.props.isOnline ? '' : 'people-item_offline';
+
+    const unreadCounter = (() => {
+      if (this.props.item.unreadMessagesCount) {
+        return (
+          <span className='threads-list__unreaded-messages'>
+            {this.props.item.unreadMessagesCount}
+          </span>
+        );
+      }
+    }());
+
     return (
       <div className={'people-item ' + offline}>
         <span className='people-item__avatar'></span>
-      {this.props.item.nick}
-      {this.props.item.unreadMessagesCount
-        ? <span className='threads-list__unreaded-messages'>{this.props.item.unreadMessagesCount}</span>
-        : null}
+        {this.props.item.nick}
+        {unreadCounter}
```

Comment:
Because i think direct chat with user will be also channel, so on "People"'s tab we will only have buttons for example to begin chat with this user, or to see his information